content/docs/custom-plugins/configuring/index.md

```diff
@@ -0,0 +1,82 @@
+---
+
+## Introduction
+
+Roadie supports all standard Backstage frontend plugin types that can be built using instructions defined in [Backstage documentation][backstage-plugin-documentation]. Your plugins can be published via npm or yarn, like publishing a normal package. We will provide you functionality to automatically add custom plugins from repositories to your Roadie instance. 
+
+If you want to have a private hosted repository for your Roadie plugins with automatic update workflow you can follow instructions on [setting up private secure Artifactory for Roadie plugins.](/docs/custom-plugins/artifactory/)
+
+## Prerequisites
+
+Custom plugins need to be enabled for your Roadie instance. You can enable this by contacting us.
+
+## Configuring a plugin for your Roadie instance
+
+### Step 1. Create plugin definition in Roadie
+
+You can navigate to h<gatsbyhack>tt</gatsbyhack>ps://your-company.roadie.so/administration/custom-plugins where you will find a collection of cards detailing custom plugins configured for the Roadie instance. 
+
+![custom_plugins_page](custom_plugins_page.png)
+
+### Form
+
+You can create a new custom plugin definition by clicking "Add New Plugin" floating button. 
+![custom_plugin_form](custom_plugin_form.png)
+
+The values that need to be added are as follows:
+
+##### Package name
+Package field corresponds to the name of the NPM package. This can be a public NPM package hosted in npmjs.com or a private package securely hosted in Roadie Artifactory. If the package is hosted in Roadie Artifactory, the name of the package needs to start with `@<your-company>-roadie/`.
+
+
+##### Plugin name
+The name of the plugin. This field can be arbitrarily chosen and will be used as a identifier when adding components from the plugin to Roadie as Cards, Tabs or Pages.
+
+
+##### Plugin version
+Semantic versioning definition for the plugin to use. This can be defined with standard NPM semantic versioning range identifiers like `^` or `~`, as well as keywords like `latest`.
+
+
+##### Components
+List of components that the plugin exposes. These are defined as a type and name pair. The type is one of Card, Content or Page. The name needs to match the component name in code that is exported from the plugin package.
+
+
+### Step 2. Installing
+
+Custom plugins are installed within the application from an external workflow. This can be triggered in a number of ways:
+1. By calling a webhook from your CI pipeline,
+2. By calling a webhook manually
+3. Automatically when a new NPM package is published to private, secure Roadie Artifactory.
+
+
+## Additional information
+
+### Plugin Statuses
+Each plugin exposes a status field indicating if they are available to be used within the Roadie instance. The statuses are as follows:
+
+#### Added
+Newly added plugin configuration. An attempt will be made to install this plugin during the next application installation cycle.
```

Comment:
I'd love a little more info about this application installation cycle. Is it a loop that checks for new plugin versions and installs them? How often does it run?

Reply:
The installation is explained in the step above, is that enough? I think we want to additionally add a documentation page on how to trigger these installations after we have confirmed the full flow

```
### Step 2. Installing

Custom plugins are installed within the application from an external workflow. This can be triggered in a number of ways:
1. By calling a webhook from your CI pipeline,
2. By calling a webhook manually
3. Automatically when a new NPM package is published to private, secure Roadie Artifactory.
```

Reply:
Ah! Sorry. I understand now. The words "application installation cycle" aren't in the section above so I didn't make the connection.

Reply:
```suggestion
Newly added plugin configuration. The new configuration will be installed when the installation workflow is triggered, as described in Step 2 above.
```